test/TestUtilities/Test.Utility/Signing/TrustedTestCert.cs

```diff
@@ -73,6 +84,57 @@ private void AddCertificateToStore()
             _store.Add(TrustedCert);
         }
 
+        //According to https://github.com/dotnet/corefx/blob/master/Documentation/architecture/cross-platform-cryptography.md#x509store,
+        //on macOS the X509Store class is a projection of system trust decisions (read-only), user trust decisions (read-only), and user key storage (read-write).
+        //So we have to run command to add certificate to System.keychain to make it trusted.
+        private void AddTrustedCert()
+        {
+            var certFile = new System.IO.FileInfo(System.IO.Path.Combine("/tmp", $"{TrustedCert.Thumbprint}.cer"));
+
+            System.IO.File.WriteAllBytes(certFile.FullName, TrustedCert.RawData);
+
+            string addToKeyChainCmd = $"sudo security add-trusted-cert -d -r trustRoot " +
+                                      $"-k \"/Library/Keychains/System.keychain\" " +
```

Comment:
When you need to use a string literal more than once, use a well-named constant instead:  `/Library/Keychains/System.keychain`

Reply:
Thanks. Fixed